client/tests/acceptance/user-can-click-rwcds-form-edit-test.js

```diff
@@ -129,32 +129,30 @@ module('Acceptance | user can click rwcds edit', function(hooks) {
 
     assert.equal(currentURL(), '/rwcds-form/1/edit');
 
+    // no-action
     await fillIn('[data-test-textarea="dcpDevelopmentsiteassumptions"]', exceedMaximum(2400, 'String'));
     assert.dom('[data-test-validation-message="dcpDevelopmentsiteassumptions"').hasText('Text is too long (max 2400 characters)');
 
-    await click('[data-test-radio="dcpExistingconditions"][data-test-radio-option="Yes"]');
+    await fillIn('[data-test-textarea="dcpDescribethenoactionscenario"]', exceedMaximum(1500, 'String'));
+    assert.dom('[data-test-validation-message="dcpDescribethenoactionscenario"').hasText('Text is too long (max 1500 characters)');
+
+    await click('[data-test-radio="dcpExistingconditions"][data-test-radio-option="No"]');
```

Comment:
Why do we click "No"  after testing the validation for  dcpDescribethenoactionscenario ? should it be before, since dcpDescribethenoactionscenario should appear regardless of whether dcpExistingconditions is yes or no?

Reply:
I might just not be understanding the form business requirements

Reply:
Correct, it does not depend on the `dcpExistingconditions` question. We click No afterwards because `dcpHowdidyoudeterminethenoactionscenario` only shows up when that radio question is No.

I put the test for all fields that are not conditional based on a question (will always be there) BEFORE we do anything with the radio buttons

Reply:
Ah okay, thanks!